### PR TITLE
Feature/bbb 686 whats your name

### PR DESCRIPTION
**Pull request by aliashik-vt**

### CODE REVIEW

File: src/main/java/uk/gov/dft/bluebadge/webapp/citizen/controllers/DeclarationSubmitController.java [hunk 1]
```diff
@@ -78,12 +78,18 @@ public String submitDeclaration(
...
     HealthConditionsForm healthConditionsForm = journey.getHealthConditionsForm();
+
     String condDesc =
         healthConditionsForm == null
             ? "Dummy condition"
             : healthConditionsForm.getDescriptionOfConditions();
 
+    String fullName = applicantNameForm == null ? "John Doe" : applicantNameForm.getFullName();
+    String birthName =
+        applicantNameForm == null ? "John Doe Birth" : applicantNameForm.getBirthName();
```
Comment (stephen-bealine) [inline src/main/java/uk/gov/dft/bluebadge/webapp/citizen/controllers/DeclarationSubmitController.java]: This is quickly going to get messy. Should we add code to bomb out if the forms aren't present?

Comment (robsvaltech) [reply src/main/java/uk/gov/dft/bluebadge/webapp/citizen/controllers/DeclarationSubmitController.java]: This is just for supporting a mock Application - someone can refactor in time. Not messy ATM..

File: src/main/java/uk/gov/dft/bluebadge/webapp/citizen/controllers/ApplicantNameController.java [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+  }
+
+  @PostMapping
+  public String submit(
+      @SessionAttribute(JOURNEY_SESSION_KEY) Journey journey,
+      @Valid @ModelAttribute("formRequest") ApplicantNameForm applicantNameForm,
+      BindingResult bindingResult,
+      Model model) {
+
+    if (!applicantNameForm.isBirthNameValid()) {
```
[the remark this answers is not in the document]
Comment (robsvaltech) [reply src/main/java/uk/gov/dft/bluebadge/webapp/citizen/controllers/ApplicantNameController.java]: Not doing this now

File: src/main/java/uk/gov/dft/bluebadge/webapp/citizen/controllers/ApplicantNameController.java [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+  private final RouteMaster routeMaster;
+
+  @Autowired
+  public ApplicantNameController(RouteMaster routeMaster) {
+    this.routeMaster = routeMaster;
+  }
+
+  @GetMapping
+  public String show(
+      @ModelAttribute("formRequest") ApplicantNameForm applicantNameForm,
```
[the remark this answers is not in the document]
Comment (robsvaltech) [reply src/main/java/uk/gov/dft/bluebadge/webapp/citizen/controllers/ApplicantNameController.java]: Not now..

**Pull request merged**